mechanical_turk/AWS_test.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
@martinbolger , @bshpringer should we still keep this file? looks like a test file to me.